setup.cfg

```diff
@@ -16,7 +16,10 @@ setup_requires = setuptools_scm
 install_requires =
     sunpy>=3
     pyvista
-
+    matplotlib
+    scipy
+    pyvirtualdisplay
```

Comment:
Similarly I don't think this should be an install requirement?

Reply:
yes, this was a venv issue sorry again